### ISSUE TITLE
WARNING: The option p will be ignored with r.

### ISSUE THREAD

**Issue by yanyew** [marbl/Mash#184]
Hello~ Thanks for developing such powerful tools.
I met the issue when I run `mash sketch -m 2 01.ccs.fq.gz -k 27 -p 5`, and the warning message is `WARNING: The option p will be ignored with r.` Is it not allow to use -p when sketch one single input?

**Comment by dayaofeijihuhuhu** [comment]
I have the same question.

**Comment by Katterinne** [comment]
Same question here.

**Comment by ondovb** [comment]
@yanyew correct, each sketch cannot be parallelized, so threads are only used for multiple sketch outputs (see #86).
